src/components/Forecast.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ForecastDay from './ForecastDay';
+
+const Forecast = props => {
+  const { apiResponse, forecastData } = props;
+  if (apiResponse === false) return <div>Loading</div>;
+  return (
+    <div className="forecast">
+      {forecastData.list.map((day, i) => (
+        <ForecastDay key={i} weather={day} isCelcius={props.isCelcius} />
```

Comment:
can you add `isCelcius` to the destructing on top?

Reply:
done